src/performance.ts

```diff
@@ -202,33 +215,46 @@ export class Performance {
   public static debug(): void {
     if (!Performance.enabled) return
 
-    const debug = require('debug')('perf')
-    debug('Total Time: %sms', Performance.highlights.runTime.toFixed(4))
```

Comment:
this was creating highlights and then letting plugin-telemetry re-massage the data.  Now we only do it once.